### PR TITLE
Fix tests

### PR DESCRIPTION
**Pull request by hanneshapke**
* updated test
* updated test requirements

### CODE REVIEW

**Comment by hanneshapke** [comment]
@tcmetzger Python 2.7 tests still fail due to test package dependencies not available under 2.7. I would suggest dropping 2.7 support with another PR.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -0,0 +1,4 @@
+:: 0.6
+
+* Updated tests, incl. complete API mocking
+* Updated test dependencies
```
Comment (tcmetzger) [inline CHANGELOG.rst:4]: This file should be merged with the existing file HISTORY.rst

**Comment by tcmetzger** [comment]
Thank you! I agree, given that Python 2.7 is officially discontinued. I would suggest merging CHANGELOG.rst with the existing file HISTORY.rst. 
I will make sure that the PyZillow version is consistently updated to V 0.6 in my upcoming pull request from the documentation branch and that supported Python versions are 3.6, 3.7, 3.8.

File: CHANGELOG.rst (hunk 1 above)
Comment (hanneshapke) [reply CHANGELOG.rst:4]: @tcmetzger Thank you for catching that. I'll update it.

Comment (tcmetzger) [reply CHANGELOG.rst:4]: Thanks!

**Pull request merged**